lua/weapons/gmod_tool/stools/wire_debugger.lua

```diff
@@ -33,6 +33,46 @@ local function IsWire(entity) --try to find out if the entity is wire
 	return false
 end
 
+properties.Add("wire_debugger", {
+	MenuLabel = "Debug",
+	MenuIcon  = "icon16/bug.png",
+
+	Filter = function(self,ent,ply)
+		if not IsValid(ent) then return false end
+		if not IsWire(ent) then return false end
+		return true
```

Comment:
This would be more readable if it was shorter. (Especially if `IsWire` also checked for validity!)